### PR TITLE
Update "internal" package dependencies as well as "external" ones

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Review by amanda-tarafa — COMMENTED** [review]
I actually wonder how does this change play along with the likes of Spanner and Diagnostics?

File: tools/Google.Cloud.Tools.ReleaseManager/GenerateProjectsCommand.cs [hunk 1]
```diff
@@ -180,22 +180,28 @@ public static void RewriteDocsRootIndex(ApiCatalog catalog)
...
+        public static void UpdateDependencies(ApiCatalog catalog, ApiMetadata api)
         {
             UpdateDependencyDictionary(api.Dependencies, "dependencies");
             UpdateDependencyDictionary(api.TestDependencies, "testDependencies");
 
             void UpdateDependencyDictionary(SortedDictionary<string, string> dependencies, string jsonName)
             {
+                // We want to update any dependencies to "external" packages as listed in DefaultPackageVersions,
+                // but also "internal" packages such as Google.LongRunning.
+                Dictionary<string, string> allDefaultPackageVersions = DefaultPackageVersions
```
Comment (amanda-tarafa) [inline tools/Google.Cloud.Tools.ReleaseManager/GenerateProjectsCommand.cs:194]: nit: Maybe do this after the if that follows, so if there are no dependencies this is not done.

**Comment by jskeet** [comment]
> I actually wonder how does this change play along with the likes of Spanner and Diagnostics?

Ooh, good point. It doesn't, because it'll try to parse "project". Will fix...

**Comment by jskeet** [comment]
Fixed - PTAL.

**Review by amanda-tarafa — APPROVED** [review]
Yep, LGTM!

File: tools/Google.Cloud.Tools.ReleaseManager/GenerateProjectsCommand.cs (hunk 1 above)
Comment (jskeet) [reply tools/Google.Cloud.Tools.ReleaseManager/GenerateProjectsCommand.cs:199]: Great minds think alike - already in the amended commit :)

**Pull request merged**